src/Microsoft.AspNetCore.Mvc.Razor.Host/Internal/GeneratedViewComponentTagHelperContext.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Razor
+{
+    /// <summary>
+    /// Contains necessary information for the view component <see cref="AspNetCore.Razor.TagHelpers.TagHelper"/> code generation process.
+    /// </summary>
+    public class GeneratedViewComponentTagHelperContext
+    {
+        /// <summary>
+        /// Instantiates a new instance of the <see cref="GeneratedViewComponentTagHelperContext"/> with default values.  
+        /// </summary>
+        public GeneratedViewComponentTagHelperContext()
+        {
+            ContextualizeMethodName = "Contextualize";
+            InvokeAsyncMethodName = "InvokeAsync";
+            IViewComponentHelperTypeName = "global::Microsoft.AspNetCore.Mvc.IViewComponentHelper";
+            IViewContextAwareTypeName = "global::Microsoft.AspNetCore.Mvc.ViewFeatures.IViewContextAware";
+            ViewContextTypeName = "global::Microsoft.AspNetCore.Mvc.Rendering.ViewContext";
+        }
+
+        /// <summary> 
+        /// Name of the Contextualize method called by an instance of the IViewContextAware type.
```

Comment:
This and the below doc comments need to use `cref` for known types. Eg: `<see cref="IViewContextAware" />`